examples/producer.java

```diff
@@ -0,0 +1,48 @@
+import java.util.concurrent.ExecutionException;
+import com.memphis.Headers;
+import com.memphis.Memphis;
+import com.memphis.MemphisConnectError;
+import com.memphis.MemphisError;
+import com.memphis.MemphisHeaderError;
+import com.memphis.MemphisSchemaError;
+import com.memphis.producer.Producer;
+
+public class Main {
+
+    public static void main(String[] args) throws ExecutionException, InterruptedException {
+        try {
+            Memphis memphis = new Memphis();
+            memphis.connect("<memphis-host>", "<application type username>", "<broker-token>").get();
+
+            Producer producer = memphis.createProducer("<station-name>", "<producer-name>");
+
+            Headers headers = new Headers();
+            headers.add("key", "value");
+
+            //Note that the produce method returns a CompletableFuture, so we need to call the get method on it to wait for the producer to finish producing the message.
+            for (int i = 0; i < 5; i++) {
+                producer.produce(
+                    ("Message #" + i + ": Hello world").getBytes("utf-8"),
+                    headers
+                ).get();
+            }
+
+        } catch (MemphisConnectError | MemphisError | MemphisHeaderError | MemphisSchemaError | InterruptedException | ExecutionException e) {
+            System.out.println(e.getMessage());
+        } finally {
+            memphis.close().get();
+        }
+    }
+}
+
+
+/** CompletableFuture - As you can see, we using a get() in the end of some rows.
```

Comment:
I don't think this lines are needed, since I guess a Java developer will probably know that. It's like explaining to a Python developer what is asyncio